File: PHP/UberBlack.php
<?php
require_once('Car.php');

class UberPool extends Car {
    public $typeCarAccepted;
    public $typeMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $typeMaterial) {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->typeMaterial = $typeMaterial;
    }
}

?>

File: JAVA/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {       /* extends means inheritance */
    Map<String, Map<String, Integer >> typeCarAccepted;      /* Map<brand, Map<model, year>>*/
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver, Map<String,
                Map<String, Integer >> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}